templates/secrets.yaml

```diff
@@ -118,6 +117,8 @@ stringData:
         # should be preserved across Dendrite restarts.
         storage_path: {{ .Values.configuration.jetstream.storage_path | quote }}
 
+        use_naffka: false
```

Comment:
Naffka isn't used anymore, with dendrite master.

Reply:
Removed. Thanks for reviewing this with sharp eyes!